src/NHSD.BuyingCatalogue.Identity.Api/ViewModels/ErrorViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NHSD.BuyingCatalogue.Identity.Api.ViewModels
```

Comment:
This view model may not even be required at this point, depending on what we decide based on the comments above.